lib/internal/streams/writable.js

```diff
@@ -53,6 +53,7 @@ const {
   getHighWaterMark,
   getDefaultHighWaterMark
 } = require('internal/streams/state');
+
```

Comment:
These newline only file changes should be removed (including the one in the test file).